MinPlatformPkg/Test/Library/TestPointCheckLib/DxeCheckMtrr.c

```diff
@@ -0,0 +1,411 @@
+/** @file
+
+Copyright (c) 2017, Intel Corporation. All rights reserved.<BR>

```

Comment:
This is authored by Intel as BSD-2-Clause-Patent but not in edk2-platforms yet?

Reply:
I changed the copyright to be Microsoft.  Intel didn't touch this code I just copied the initial header.